[2022-09-13 00:39:17,893] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.choose_model 2022-09-12T00:00:00+00:00 [queued]>
[2022-09-13 00:39:18,083] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.choose_model 2022-09-12T00:00:00+00:00 [queued]>
[2022-09-13 00:39:18,084] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 00:39:18,084] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-09-13 00:39:18,085] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 00:39:18,127] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): choose_model> on 2022-09-12T00:00:00+00:00
[2022-09-13 00:39:18,132] {standard_task_runner.py:52} INFO - Started process 27680 to run task
[2022-09-13 00:39:18,163] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'choose_model', '2022-09-12T00:00:00+00:00', '--job-id', '34', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpvk8ja2tc', '--error-file', '/tmp/tmp4vr0c4wk']
[2022-09-13 00:39:18,166] {standard_task_runner.py:77} INFO - Job 34: Subtask choose_model
[2022-09-13 00:39:18,277] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_dag.choose_model 2022-09-12T00:00:00+00:00 [running]> on host airflowvm
[2022-09-13 00:39:18,476] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=choose_model
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T00:00:00+00:00
[2022-09-13 00:39:18,497] {python.py:151} INFO - Done. Returned value was: None
[2022-09-13 00:39:18,523] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=choose_model, execution_date=20220912T000000, start_date=20220913T003917, end_date=20220913T003918
[2022-09-13 00:39:18,567] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-13 00:39:18,583] {local_task_job.py:151} INFO - Task exited with return code 0
